core/templates/dev/head/pages/exploration_player/exploration_footer_directive.html

```diff
@@ -114,14 +114,42 @@
     float: left;
   }
 
+  .oppia-exploration-footer .oppia-navbar-footer-info-icon {
+    height: 56px;
+    padding: 12px;
+  }
+
+  .oppia-exploration-footer .oppia-navbar-footer-info-icon:hover {
+    background: #fff;
+    color: #009688;
+    height: 56px;
+    padding: 12px;
+  }
+
+  .oppia-exploration-footer .modal-dialog {
```

Comment:
It turns out that the modal-dialog is appended to the body, so this particular style can't have a prefix. Let's delete this style from here and put the max-width in the existing `.oppia-modal-information-card .modal-dialog {` block in oppia.css.